src/coreclr/src/vm/jitinterface.cpp

```diff
@@ -12975,6 +12993,9 @@ PCODE UnsafeJitFunction(PrepareCodeConfig* config,
         printf(".");
 #endif // _DEBUG
 
+    FastInterlockExchangeAddLong((LONG64*)&g_cbILJitted, methodInfo.ILCodeSize);
+    FastInterlockIncrement((LONG*)&g_cMethodsJitted);
```

Comment:
It would be nice to delete `g_JitCount` static in this file because of it counts the exact same thing now.